src/main/java/edu/harvard/iq/dataverse/DatasetField.java

```diff
@@ -262,7 +264,7 @@ public String getCompoundDisplayValue() {
             for (DatasetField dsf : dscv.getChildDatasetFields()) {
                 for (String value : dsf.getValues()) {
                     if (!(value == null)) {
-                        returnString += (returnString.equals("") ? "" : "; ") + value.trim();
+                        returnString += (returnString.isEmpty() ? "" : "; ") + value.trim();
```

Comment:
Good point, I had overlooked that. Maybe I hoped that `value` won't be `null`… :)